internal/primitive/transform/action/strings/extract_between_delimiters_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Linkall Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strings_test
+
+import (
+	"testing"
+
+	cetest "github.com/cloudevents/sdk-go/v2/test"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/action/strings"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/context"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/runtime"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestExtractBetweenDelimitersAction(t *testing.T) {
+	funcName := strings.NewExtractBetweenDelimitersAction().Name()
+	Convey("test Positive testcase", t, func() {
```

Comment:
can you add some cases:
1. start delimiter equals end delimiter and the original string only has one delimiter, such as `Hi welcome to &vanus`, the start delimiter and end delimiter is `&`